apps/lwipserver2/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+cmake_minimum_required(VERSION 3.8.2)
+
+project(lwipserver_single C)
+includeGlobalComponents()
+
+find_package(projects_libs REQUIRED)
+
+set(CAmkESCPP ON CACHE BOOL "" FORCE)
+if("${KernelArch}" STREQUAL "x86")
+    set(cpp_define -DKernelArchX86)
+elseif("${KernelArch}" STREQUAL "arm")
+    set(cpp_define -DKernelArchArm)
+endif()
+
+include(${LWIP_HELPERS})
+AddLWIPConfiguration(${CMAKE_CURRENT_LIST_DIR}/lwip_include)
+
+set(libs sel4utils sel4vka sel4allocman sel4vspace sel4simple 
+    sel4platsupport lwip)
+
+set(sources src/tcp_echo_socket.c src/udp_echo_socket.c src/utilization_socket.c)
+
+DeclareCAmkESComponent(
+    LWIPServer
+    SOURCES
+    ${sources}
+    INCLUDES
+    include
+    LIBS
+    ${libs}
+    LD_FLAGS
+    -Wl,--section-start=.note.gnu.build-id=0x400920
```

Comment:
It this just a copy paste from somewhere? Maybe add a comment about this, that this is just optional to set a specific build ID. The kernel is using `-Wl,--build-id=none" # Ensure reproducible builds` here.